sdk/lib/opentelemetry/sdk/trace/export/batch_span_processor.rb

```diff
@@ -47,7 +47,8 @@ def initialize(exporter:,
                          exporter_timeout_millis: Float(ENV.fetch('OTEL_BSP_EXPORT_TIMEOUT_MILLIS', 30_000)),
                          schedule_delay_millis: Float(ENV.fetch('OTEL_BSP_SCHEDULE_DELAY_MILLIS', 5_000)),
                          max_queue_size: Integer(ENV.fetch('OTEL_BSP_MAX_QUEUE_SIZE', 2048)),
-                         max_export_batch_size: Integer(ENV.fetch('OTEL_BSP_MAX_EXPORT_BATCH_SIZE', 512)))
+                         max_export_batch_size: Integer(ENV.fetch('OTEL_BSP_MAX_EXPORT_BATCH_SIZE', 512)),
+                         start_thread_on_boot: OpenTelemetry::SDK::Internal.to_boolean(ENV['OTEL_START_EXPORT_THREAD_ON_BOOT']))
```

Comment:
[The spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md#language-specific-environment-variables) suggest this env var should have an `OTEL_RUBY_` prefix. Given it applies to the `BatchSpanProcessor`, we should pattern it after [these env vars](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md#batch-span-processor), so `OTEL_RUBY_BSP_START_EXPORT_THREAD_ON_BOOT`.

Reply:
I've gone with `OTEL_BSP_START_THREAD_ON_BOOT` to be congruent the other vars.